optuna/integration/allennlp/_pruner.py

```diff
@@ -155,6 +152,8 @@ def __init__(
                 "please install Optuna v2.5.0 by executing `pip install 'optuna==2.5.0'`."
             )
 
+        self.pruned = False
```

Comment:
We don't need this anymore?

Reply:
You're right, removed in 99c91bc. :pray: